src/main/java/org/jboss/arquillian/extension/jacoco/client/configuration/JaCoCoConfiguration.java

```diff
@@ -94,4 +94,13 @@ private static boolean hasClassJacocoAccessField(Class<?> clazz) {
         }
         return true;
     }
+
+    private static boolean hasClass(String className) {
+        try {
+            Class.forName(className);
+        } catch (Exception e) {
```

Comment:
How about narrowing it to `ClassNotFoundException` and fail with other exceptions?